cmd/bbolt/command_surgery_cobra.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"go.etcd.io/bbolt/internal/common"
+	"go.etcd.io/bbolt/internal/surgeon"
+)
+
+var (
+	surgeryTargetDBFilePath string
+	surgeryPageId           uint64
+	surgeryStartElementIdx  int
+	surgeryEndElementIdx    int
+)
+
+func newSurgeryCobraCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "surgery <subcommand>",
+		Short: "surgery related commands",
+	}
+
+	cmd.AddCommand(newSurgeryClearPageElementsCommand())
+
+	return cmd
+}
+
+func newSurgeryClearPageElementsCommand() *cobra.Command {
+	clearElementCmd := &cobra.Command{
+		Use:   "clear-page-elements [options]",
```

Comment:
I assume it's `clear-page-elements [bbolt-file] [options]`

Reply:
Updated.
```
clear-page-elements <bbolt-file> [options]
```